templates/includes/sidebar.html

```diff
@@ -1,6 +1,32 @@
 <ul class="sidebar-nav">
-  <li><a href="{% url "taxi:index" %}">Home page</a></li>
-  <li><a href="{% url "taxi:manufacturer-list" %}">Manufacturers</a></li>
-  <li><a href="{% url "taxi:car-list" %}">Cars</a></li>
-  <li><a href="{% url "taxi:driver-list" %}">Drivers</a></li>
+  {% if user.is_authenticate %}
```

Comment:
There is a typo in the template tag. It should be `{% if user.is_authenticated %}` instead of `{% if user.is_authenticate %}`. This will correctly check if the user is authenticated.